src/NavBar.ts

```diff
@@ -7,6 +7,8 @@ import { window } from '@/Window'
 import QueryStore, { QueryPoint, QueryPointType, QueryStoreState } from '@/stores/QueryStore'
 import MapOptionsStore, { MapOptionsStoreState, StyleOption } from './stores/MapOptionsStore'
 
+let fakeNavi: boolean = false
+
```

Comment:
Export was a useless leftover mistake

Reply:
Sorry, I didn't mean the visibility. I mean the nature that an unrelated call of parseURL could change fakeNavi. I know it is rather nit-picky but this will make it easier to get the experimental navi branch or parts of it merged into master.